Documentation/ApiOverview/Seo/PageTitleApi.rst

```diff
@@ -147,3 +147,9 @@ First the :php:`SeoTitlePageTitleProvider` (because it will be handled before
 You can override these settings within your own installation. You can add as
 many providers as you want. Be aware that if a provider returns a non-empty
 value, all provider with a lower priority will not be checked.
+
+
+See also
```

Comment:
We have also a `.. seealso::` admonition, this may be better suited. What do you think?